app/assets/javascripts/short-form/components/preferenceProofUploader.js.coffee

```diff
@@ -72,6 +72,10 @@ angular.module('dahlia.components')
           opts = @rentBurdenOpts()
         FileUploadService.uploadProof($file, @preference, @listingId, opts).then =>
           @afterUpload()
+        if @preference == 'neighborhoodResidence' || @preference == 'antiDisplacement'
+          # if we're uploading for NRHP/ADHP, it also copys info and uploads for liveInSf so that the file info is saved into DB
+          ShortFormApplicationService.copyNeighborhoodToLiveInSf(@preference)
```

Comment:
@kaptron I had to call this function on upload in order to get the file info saved into the database. Before this, we only transfer the info during `copyNeighborhoodToLiveInSf` which is called as part of the callback `checkAfterLiveInTheNeighborhood` when you click next (on ADHP/DTHP page). Now that this order of operations makes more sense, I wonder if we even need to called `copyNeighborhoodToLiveInSf` in the callback. My sense is no, so I want to remove it. Thoughts?